pkg/util/async/bounded_frequency_runner_test.go

```diff
@@ -61,6 +61,7 @@ type fakeTimer struct {
 
 func newFakeTimer() *fakeTimer {
 	ft := &fakeTimer{
+		now:     time.Date(2000, 0, 0, 0, 0, 0, 0, time.UTC),
```

Comment:
workaround for https://github.com/golang/go/issues/23482 when used with a fake clock starting at time 0